src/cartservice/docker-compose.yaml

```diff
@@ -23,6 +23,5 @@ services:
       - "7070:7070"
     environment:
       REDIS_ADDR: "redis:6379"
-      ASPNETCORE_URLS: "http://0.0.0.0:7070"
```

Comment:
why was this removed?

Reply:
This environment variable is now part of the `Dockerfile`, so no need to add it here.